react/react1/week1/todo/src/components/DeadlineDate.js

```diff
@@ -0,0 +1,19 @@
+function DeadlineDate(props) {
+  console.log("propsÆ ", props.date);
+  const propsDate = props.date.split(" ");
+  console.log(propsDate);
+  const weekday = propsDate[0];
+  const month = propsDate[1];
+  const day = propsDate[2];
+  const year = propsDate[3];
```

Comment:
These are very static. You need to use JavaScript date methods, such as `getDate()`